### PR TITLE
matching release notes dates

### PR DESCRIPTION
**Pull request by jameshupp**
This is so the email and web dates match.

### CODE REVIEW

File: content/updates/2017-05-22-release-notes.md
Comment (khandelwal) [inline content/updates/2017-05-22-release-notes.md]: I know it doesn't fit with the style of the other headings but renaming this to "Volunteers needed" or something like that it will help highlight the call to action.

File: content/updates/2017-05-22-release-notes.md
Comment (khandelwal) [inline content/updates/2017-05-22-release-notes.md]: Here's another call to action that should probably be better highlighted.

File: content/updates/2017-05-22-release-notes.md
Comment (khandelwal) [inline content/updates/2017-05-22-release-notes.md]: And here too, is another call to action.

**Review by khandelwal — CHANGES_REQUESTED** [review]

File: content/updates/2017-05-22-release-notes.md
Comment (jameshupp) [reply content/updates/2017-05-22-release-notes.md]: I think I'm going to go with `Help make cloud.gov easier to use` since it's pretty directly for users who have a vested interest in that.

Comment (jameshupp) [reply content/updates/2017-05-22-release-notes.md]: n/m looks like @mogul added the other version, which is sufficient for me

**Review by mogul — DISMISSED** [review]

File: content/updates/2017-05-22-release-notes.md
Comment (jameshupp) [reply content/updates/2017-05-22-release-notes.md]: Unlike the volunteer ask, this is something we've done before and used this style. I think it's fine to work on this—I'll add it to the release notes template—but I don't want to hold up publishing this edition for it unless you think it's essential.

File: content/updates/2017-05-22-release-notes.md
Comment (mogul) [reply content/updates/2017-05-22-release-notes.md]: Only because I was faster on the draw... Make it say what you think it should say!

File: content/updates/2017-05-22-release-notes.md
Comment (jameshupp) [reply content/updates/2017-05-22-release-notes.md]: Same as above. One thing we could do, though, is **use strong text on the CTAs**

Comment (mogul) [reply content/updates/2017-05-22-release-notes.md]: That was my first thought for how to address it, and seems like a good interim step to unblock shipping this one.

**Review by khandelwal — APPROVED** [review]

**Pull request merged**
